device-types/Cisco/Meraki-MS225-48LP.yaml

```diff
@@ -5,11 +5,16 @@ slug: ms225-48lp
 part_number: MS225-48LP-HW
 u_height: 1
 is_full_depth: false
+comments: '[MS225 Overview and Specifications](https://meraki.cisco.com/lib/pdf/meraki_datasheet_ms225.pdf)'
 power-ports:
   - name: Slot 1
     type: iec-60320-c14
     maximum_draw: 490
     allocated_draw: 53
+  - name: Cisco RPS 2300 connector for PSU failover
```

Comment:
The name on this should be either what appears in  the config or printed on the device.. this name is a little too long to make sense

Reply:
@danner26 I updated the name as to what's written in the documentation.  Unfortunately it's not a whole lot shorter.  It has no physical label on the device.

Reply:
Why dont we call it something like `RPScon` then, its shorter and to the point that way

Reply:
Updated as "RPS Connector" which has some basis in technical documentation and is more brief that than the other options.